app/components/works/show/files_component.html.erb

```diff
@@ -14,6 +14,7 @@
   </tr>
   </thead>
   <tbody>
-    <%= render Works::Show::AttachedFileComponent.with_collection(attached_files, work_version: work_version) %>
+    <%= render Works::Show::AttachedFileComponent.with_collection(attached_files.sort_by { |af| af.path.to_s.downcase },
```

Comment:
1. `.path` is already a string, so `.to_s` isn't necessary.
2. Slight preference for `attached_file` instead of `af`.

Reply:
Got it. Have made both of those changes.